### PR TITLE
applied orderby to nod list

### PR DESCRIPTION
**Pull request by matbusby-fw**
## Objective 

[MDS-4264](https://bcmines.atlassian.net/browse/MDS-4264)

applied order_by to the fetching of the nod list (sorting by `create_timestamp` descending)

### CODE REVIEW

File: services/core-api/app/api/mines/notice_of_departure/models/notice_of_departure.py [hunk 1]
```diff
@@ -85,13 +86,13 @@ def find_one(cls, __guid, include_documents=False):
 
     @classmethod
     def find_all_by_mine_guid(cls, __guid):
-        return cls.query.filter_by(mine_guid=__guid, deleted_ind=False).all()
+        return cls.query.filter_by(mine_guid=__guid, deleted_ind=False).order_by(NoticeOfDeparture.create_timestamp.desc()).all()
```
Comment (hitankar-freshworks) [inline services/core-api/app/api/mines/notice_of_departure/models/notice_of_departure.py]:
```suggestion
        return cls.query.filter_by(mine_guid=__guid, deleted_ind=False).order_by(cls.create_timestamp.desc()).all()
```

File: services/core-api/app/api/mines/notice_of_departure/models/notice_of_departure.py [hunk 2]
```diff
@@ -85,13 +86,13 @@ def find_one(cls, __guid, include_documents=False):
...
 
     @classmethod
     def find_all_by_permit_guid(cls, __guid, mine_guid=None):
-        query = cls.query.filter_by(permit_guid=__guid, deleted_ind=False)
+        query = cls.query.filter_by(permit_guid=__guid, deleted_ind=False).order_by(NoticeOfDeparture.create_timestamp.desc())
```
Comment (hitankar-freshworks) [inline services/core-api/app/api/mines/notice_of_departure/models/notice_of_departure.py]:
```suggestion
        query = cls.query.filter_by(permit_guid=__guid, deleted_ind=False).order_by(cls.create_timestamp.desc())
```

File: services/core-api/app/api/mines/notice_of_departure/models/notice_of_departure.py [hunk 3]
```diff
@@ -85,13 +86,13 @@ def find_one(cls, __guid, include_documents=False):
...
         if mine_guid:
-            query = cls.query.filter_by(permit_guid=__guid, mine_guid=mine_guid, deleted_ind=False)
+            query = cls.query.filter_by(permit_guid=__guid, mine_guid=mine_guid, deleted_ind=False).order_by(NoticeOfDeparture.create_timestamp.desc())
```
Comment (hitankar-freshworks) [inline services/core-api/app/api/mines/notice_of_departure/models/notice_of_departure.py]:
```suggestion
            query = cls.query.filter_by(permit_guid=__guid, mine_guid=mine_guid, deleted_ind=False).order_by(cls.create_timestamp.desc())
```

**Review by hitankar-freshworks — DISMISSED** [review]
I think we can use `cls` of the actual name. Approving it but please make the necessary changes.

File: services/core-api/app/api/mines/notice_of_departure/models/notice_of_departure.py (hunk 1 above)
Comment (matbusby-fw) [reply services/core-api/app/api/mines/notice_of_departure/models/notice_of_departure.py]: Okay, cool!  Makes sense.

**Review by hitankar-freshworks — APPROVED** [review]

**Pull request merged**